### PR TITLE
Took a pass at challenges

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: vac-comm.md [hunk 1]
```diff
@@ -4,12 +4,21 @@
...
-- scalability of the network
-- incentived infrastructure and spam-resistance
-- build with resource restricted devices in mind, including nodes being mostly offline
+@TODO
+
+### Challenges
+
+The current challenges we experience with private and decentralized communication are often related to the fact that we are building a decentralized protocol and not one which is centralized. This one property, although large adds quite a few complexities to the protocol which we are working on resolving.
+
+Firstly, scalability of the network becomes hard. When a centralized chat service has problems handling its DAU, we simply add a new server. This is easier said than done with a decentralized protocol, adding a new node does not guarantee increase of DAU.
```
Comment (oskarth) [inline vac-comm.md]: Too chat centric?

File: vac-comm.md [hunk 2]
```diff
@@ -4,12 +4,21 @@
...
 
-**Challenges being addressed:**
-- scalability of the network
-- incentived infrastructure and spam-resistance
-- build with resource restricted devices in mind, including nodes being mostly offline
+@TODO
+
+### Challenges
+
+The current challenges we experience with private and decentralized communication are often related to the fact that we are building a decentralized protocol and not one which is centralized. This one property, although large adds quite a few complexities to the protocol which we are working on resolving.
```
Comment (oskarth) [inline vac-comm.md]: Feels a bit wordy, maybe shorter sentences?

File: vac-comm.md [hunk 3]
```diff
@@ -4,12 +4,21 @@
...
+
+Firstly, scalability of the network becomes hard. When a centralized chat service has problems handling its DAU, we simply add a new server. This is easier said than done with a decentralized protocol, adding a new node does not guarantee increase of DAU.
+
+In a centralized application spam-resistance is easy to handle, we can simply throttle one user and be done with it, this however is not the case with decentralized messaging protocols. Additionally, currently whisper uses proof-of-work to reduce spam, which it does not really do as we need to lower the PoW so far for it to be mobile friendly that it is easy to spam the network yet again.
```
Comment (oskarth) [inline vac-comm.md]: Mention amplification? Also shorter sentences for readability

File: vac-comm.md [hunk 4]
```diff
@@ -4,12 +4,21 @@
...
-## Motivation and challenges
+## Motivation 
 
-**Challenges being addressed:**
-- scalability of the network
-- incentived infrastructure and spam-resistance
-- build with resource restricted devices in mind, including nodes being mostly offline
+@TODO
+
+### Challenges
```
Comment (oskarth) [inline vac-comm.md]: Can we make this more skimmable? E.g. first mention 3 challenges being addressed, then have a paragraph each for them

File: vac-comm.md [hunk 5]
```diff
@@ -4,12 +4,21 @@
...
 vac.comm provides private and decentralized communication, and is part of the Vac communication/storage/compute umbrella.
 
-## Motivation and challenges
+## Motivation 
 
-**Challenges being addressed:**
-- scalability of the network
-- incentived infrastructure and spam-resistance
-- build with resource restricted devices in mind, including nodes being mostly offline
+@TODO
```
Comment (oskarth) [inline vac-comm.md]: Maybe use decentralized brief para or something from Vac rough overview post?

**Comment by oskarth** [comment]
Ping @decanus

**Comment by decanus** [comment]
ah yes will get back to this, forgot the pr. Thanks for ping @oskarth

File: vac-comm.md (hunk 5 above)
Comment (oskarth) [reply vac-comm.md]: I meant pasting from https://vac.dev/vac-overview, realize it wasn't elaborated on too much

File: vac-comm.md [hunk 6]
```diff
@@ -4,12 +4,25 @@
...
+
+**Scalability** is a large challenge we face. When a centralized messaging protocol has problems handling its load, we simply add a new server. This is easier said than done with a decentralized protocol, adding a new node does not guarantee increase of throughput.
+
+**Spam-resistance** is another, in a centralized application spam-resistance is easy to handle, we can simply throttle one user and be done with it. This however is not the case with decentralized messaging protocols. Additionally, currently whisper uses proof-of-work to reduce spam, which does not actually prevent spam as we need to lower the PoW so far for it to be mobile friendly that it is easy to spam the network yet again.
+
+In order to add a large amount of participants actively supporting the network, we need to somehow incentivize those users.
+
+**Resource restricted devices** are our main user base. Nodes are mostly offline, which is not an issue for a centralized messaging protocol where all messages are stored on a server to later be downloaded by a client in order to read. How do we guarantee that messages reach their destination even if it is mostly offline?
```
Comment (oskarth) [inline vac-comm.md]: Thats one resource restriction, also BW and lack of public IP
